cli/cmd/generate_execute.go

```diff
@@ -370,30 +370,33 @@ func TerraformPlanAndExecute(workingDir string) error {
 		return err
 	}
 
-	// Write plan
-	changes, err := TerraformExecPlan(tf)
-	if err != nil {
-		return err
-	}
+	// If running in nonInteractive mode, dont run a plan or prompt to proceed; just proceed
+	if !cli.nonInteractive {
```

Comment:
could we please use the function, rather than one single variable?
```suggestion
	if !cli.InteractiveMode() {
```

Reply:
👍